### PR TITLE
#8 add simple maven cucumber example

### PR DESCRIPTION
**Pull request by joergsesterhenn**
This is one of my first PRs so please let me know if anything needs to be changed 😉

### CODE REVIEW

File: examples/maven/pom.xml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+			<version>${junit.version}</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>com.aventstack</groupId>
+			<artifactId>extentreports</artifactId>
+			<version>${extentreports.version}</version>
+			<scope>provided</scope>
+		</dependency>
+	</dependencies>
```
Comment (haroon-sheikh) [inline examples/maven/pom.xml]: Can we remove any dependencies that are not being used?

File: examples/maven/pom.xml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+		<java.version>1.8</java.version>
+		<extentreports.version>3.0.7</extentreports.version>
+		<cucumber.version>1.2.5</cucumber.version>
+		<junit.version>4.12</junit.version>
+		<maven-compiler-plugin.version>3.7.0</maven-compiler-plugin.version>
+		<maven-source-plugin.version>3.0.1</maven-source-plugin.version>
+		<maven-javadoc-plugin.version>2.10.4</maven-javadoc-plugin.version>
```
Comment (haroon-sheikh) [inline examples/maven/pom.xml]: Remove unused properties

File: examples/maven/pom.xml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+			<url>http://www.opensource.org/licenses/mit-license.php</url>
+			<distribution>repo</distribution>
+		</license>
+	</licenses>
+
+	<dependencies>
+    <dependency>
+      <groupId>com.sitture</groupId>
+      <artifactId>cucumber-jvm-extentreport</artifactId>
+      <version>2.0.3-SNAPSHOT</version>
```
Comment (haroon-sheikh) [inline examples/maven/pom.xml]: lets move this version into properties?

**Review by haroon-sheikh — CHANGES_REQUESTED** [review]
Hi @joergsesterhenn, thanks a ton for your contribution. Just a few minor changes to make please.

**Comment by joergsesterhenn** [comment]
I hope this is acceptable?

**Review by haroon-sheikh — APPROVED** [review]

**Pull request merged**